tests/safeds/data/tabular/transformation/test_imputer.py

```diff
@@ -280,7 +288,13 @@ def test_should_return_transformed_table(
         strategy: ImputerStrategy,
         expected: Table,
     ) -> None:
-        assert Imputer(strategy).fit_and_transform(table, column_names) == expected
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                action="ignore",
+                message=r"There are multiple most frequent values in a column given to the Imputer\..*",
+                category=UserWarning,
+            )
+            assert Imputer(strategy).fit_and_transform(table, column_names) == expected
```

Comment:
```suggestion
        if isinstance(strategy, Imputer.Strategy.Mode):
            with warnings.catch_warnings():
                warnings.filterwarnings(
                    action="ignore",
                    message=r"There are multiple most frequent values in a column given to the Imputer\..*",
                    category=UserWarning,
                )
                assert Imputer(strategy).fit_and_transform(table, column_names) == expected
        else:
            assert Imputer(strategy).fit_and_transform(table, column_names) == expected
```
Only filter the warning if you use the `Mode` strategy